### PR TITLE
added monastary

### PR DESCRIPTION
**Pull request by Xavier-J-Ortiz**
Refs:https://github.com/openstreetmap/iD/issues/4932

### CODE REVIEW

**Comment by Xavier-J-Ortiz** [comment]
@bhousel I left the icon as `place-of-worship` since I'm not sure if a `monastery-grounds`icon exists.

Let me know if this looks good.

File: data/presets/presets.json [hunk 1]
```diff
@@ -1955,6 +1955,51 @@
...
         },
+        "amenity/monastery": {
+            "icon": "place-of-worship",
+            "fields": [
+                "name",
+                "religion",
+                "denomination",
+                "address",
+                "building_area",
+                "service_times"
```
Comment (bhousel) [inline data/presets/presets.json]: I don't think they do services at monasteries.  Can you remove this field?  Everything else about this looks great 👍

**Comment by Xavier-J-Ortiz** [comment]
Oh right!

Changed, committed, and pushed.

**Comment by bhousel** [comment]
Thanks @Xavier-J-Ortiz !

**Comment by alexander-ii** [comment]
It needs to be mentioned, that there is possibly a mistake. This PD [introduces a new tag](https://github.com/openstreetmap/iD/pull/4936/commits/790740dac52fe45a5be5315899db8a2419df695a#diff-7337e7ffe7c0b25dad3b0305cfca8b0bR42) `amenity=monastery_grounds`. Which is absent in [wiki](https://wiki.openstreetmap.org/w/index.php?search=monastery_grounds&title=Special:Search&go=%D0%9F%D0%B5%D1%80%D0%B5%D0%B9%D1%82%D0%B8&searchToken=3ch645ujq594rs79ve9kxdbxj) and [taginfo](https://taginfo.openstreetmap.org/search?q=monastery_grounds#values) as well.

Please, fix it.

**Pull request merged**